Extensions/ExtendedVariables.json

```diff
@@ -19,7 +19,7 @@
   ],
   "eventsFunctions": [
     {
-      "description": "A global variable exist.",
+      "description": "Check if a global variable exists.",
       "fullName": "Global variable exist",
```

Comment:
I think this should be `Global variable exists` actually for the name too (and the scene/object variable).